bodhi/client/__init__.py

```diff
@@ -101,6 +103,21 @@ def _add_options(func):
     return _add_options
 
 
+def handle_errors(method):
+    """ A decorator that echoes neat error messages on AuthError or BodhiClientException. """
+    @functools.wraps(method)
+    def wrapper(*args, **kwargs):
+        try:
+            method(*args, **kwargs)
+        except AuthError as e:
+            click.echo("%s: Check your FAS username & password" % (e))
+            sys.exit(1)
+        except bindings.BodhiClientException as e:
+            click.echo(str(e))
+            sys.exit(-1)
```

Comment:
Technically, the exit code is an unsigned int. Though this does not cause a real failure, the resulting exit code is also not -1:

```
$ python -c "import sys; sys.exit(-1)"
$ echo $?
255
```

If you want to use a unique exit code here, you could just pick any unused code. I think only 1 is used here, so you could use 2.